deploy-board/deploy_board/static/js/components/capacitycomponents.js

```diff
@@ -313,8 +327,17 @@ Vue.component("static-capacity-config", {
                     this.sizeWarning = getCapacityScaleDownAlertMessage(true, true);
                 }
             }
+
             this.imbalanceWarning = checkImbalance(this.placements, calculateImbalanceThreshold(sizeIncrease, this.placements.length));
             this.showImbalanceWarning = this.imbalanceWarning != '';
+
+            if ((!(this.terminationLimit === null)) && (-sizeIncrease > this.terminationLimit)) {
```

Comment:
Maybe extract the common part out
```
if ( this.terminationLimit !== null) {
...
}
```

Reply:
+1 on using `this.terminationLimit !== null ` instead of `!(this.terminationLimit === null)`

Reply:
updated

Reply:
This isn't fully resolved.

Reply:
done